### PR TITLE
Support test cases

### PR DESCRIPTION
**Pull request by vikasranatalentelgia**
Support test cases

### CODE REVIEW

File: src/components/ComponentPages/DelegatedSupportCamps/__test__/delegatedSupportCamps.test.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ describe("Delegated Support camps page", () => {
     );
     expect(getByText("Remove")).toBeTruthy();
     expect(getByText("Cancel")).toBeTruthy();
-    expect(screen.getByText("Remove Support")).toBeInTheDocument();
+    expect(screen.getAllByText("Remove Support")).toBeTruthy();
```
Comment (akuks) [inline src/components/ComponentPages/DelegatedSupportCamps/__test__/delegatedSupportCamps.test.tsx]: Please move "Remove Support" to common message or label file

File: src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx [hunk 2]
```diff
@@ -195,7 +195,7 @@ describe("Direct Support camps page", () => {
       />
     );
     expect(screen.getByText("Remove")).toBeTruthy();
-    expect(screen.getByText("Remove Support")).toBeInTheDocument();
+    expect(screen.getAllByText("Remove Support")).toBeTruthy();
```
Comment (akuks) [inline src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx]: Please move "Remove Support" to common message or label file

File: src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx [hunk 3]
```diff
@@ -195,7 +195,7 @@ describe("Direct Support camps page", () => {
       />
     );
     expect(screen.getByText("Remove")).toBeTruthy();
```
Comment (akuks) [inline src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx]: Please move "Remove" to common message or label file

**Review by akuks — CHANGES_REQUESTED** [review]
@vikasranatalentelgia Please do the requested changes

File: src/components/ComponentPages/DelegatedSupportCamps/__test__/delegatedSupportCamps.test.tsx (hunk 1 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DelegatedSupportCamps/__test__/delegatedSupportCamps.test.tsx]: Done

File: src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx (hunk 3 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx]: Done

File: src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx (hunk 2 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx]: Done

**Review by akuks — APPROVED** [review]

**Pull request merged**